Library/src/Web/EmscriptenWebClient/EmscriptenWebClient.cpp

```diff
@@ -14,11 +14,12 @@
  * limitations under the License.
  */
 
-#include "EmscriptenWebClient.h"
+//#ifdef CSP_WASM
 
 #include "CSP/Common/Map.h"
 #include "CSP/Common/String.h"
 #include "Debug/Logging.h"
+#include "EmscriptenWebClient.h"
```

Comment:
This should be the first include. If clang-format is moving it, put a new line after it.